docs/docs/licensing.md

```diff
@@ -15,7 +15,7 @@ The current limitations are applied to the community version:
 
 !!! Note "Accepting the EULA"
     You need to pass in a setting to Qlik Associative Engine to acknowledge that
-    you accept the EULA. Example: `docker run qlikcore/engine -S AcceptEULA=yes`
+    you accept the EULA. Example: `docker run qlikcore/engine:<version> -S AcceptEULA=yes`
```

Comment:
Maybe should be `Tag Name` instead of `version`?
Tag Name is used on the dockerhub page
https://hub.docker.com/r/qlikcore/engine/tags/

Reply:
sounds good, and maybe be consistent by changing this everywhere, here:
https://github.com/qlik-oss/core-website/blob/master/docs/docs/services/qix-engine/access-control.md#configuration && here => https://github.com/qlik-oss/core-website/blob/master/docs/docs/services/qix-engine/doc-synchronization.md#configuration ?